Pages/wishlist.html

```diff
@@ -0,0 +1,151 @@
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star-half-alt"></i>
+                </div>
+                <a href="#" class="btn">Buy Now</a>
+            </div>
+    
+            <div class="box">
+                <img src="../image/s-img-2.jpg" alt="Pizza">
+                <h3>Pizza</h3>
+                <div class="price"> $4.99/- -10.99/-</div>
+                <div class="stars">
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star-half-alt"></i>
+                </div>
+                <a href="#" class="btn">Buy Now</a>
+            </div>
+    
+            <div class="box">
+                <img src="../image/s-img-3.jpg" alt="ice-cream">
+                <h3>Ice-cream</h3>
+                <div class="price"> $4.99/- -10.99/-</div>
+                <div class="stars">
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star-half-alt"></i>
+                </div>
+                <a href="#" class="btn">Buy Now</a>
+            </div>
+
+            <div class="box">
+                <img src="../image/s-img-4.jpg" alt="ice-cream">
+                <h3>Cold Drink</h3>
+                <div class="price"> $4.99/- -10.99/-</div>
+                <div class="stars">
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star"></i>
+                    <i class="fas fa-star-half-alt"></i>
+                </div>
+                <a href="#" class="btn">Buy Now</a>
+            </div>
+        </div>
+    
+    </section>
+    
+    <!-- categories section ends -->
+
+    <!-- Footer section starts -->
+    <section class="footer">
+    
+        <div class="box-container">
+    
+            <div class="box">
+                <h3> Wow Shop <i class="fas fa-shopping-basket"></i> </h3>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+                <div class="share">
+                    <a href="#" class="fab fa-facebook-f"></a>
+                    <a href="#" class="fab fa-twitter"></a>
+                    <a href="#" class="fab fa-instagram"></a>
+                    <a href="#" class="fab fa-linkedin"></a>
+                </div>
+            </div>
+    
+            <div class="box">
+                <h3>contact info</h3>
+                <a href="#" class="links"> <i class="fas fa-phone"></i> +123-456-7890 </a>
```

Comment:
Flip the phone icon. You can add class `fa-rotate-90` or `fa-flip-horizontal` to the icon

Reply:
Okayy!!